### DISCUSSION TITLE
Jira tasks integration (in addition to backlog items)

### DISCUSSION THREAD

**Discussion by jordanh** [ParabolInc/parabol#5299] [Feedback]
Today our Jira integration supports import/export of backlog tickets, but a couple of users in a recent user research survey of ours have asked for Jira tasks to sync as well.

With out new integrations architecture, this should be easier.

**Comment by mattkrick** [comment]
Clarifying Q:
- Does this mean we only sync `stories` but not `tasks`? Or Does this mean we're just not grabbing every single jira issue?
